screens/SendFeedbackScreen.tsx

```diff
@@ -134,12 +137,30 @@ export default function SendFeedbackScreen({ navigation }) {
       base64: true,
       quality: 0.5,
     });
-    if (!result.cancelled && result !== null) {
-      setSelectImage(true);
-      setImage("data:image/jpeg;base64," + result["base64"]);
+    if (!result.canceled && result !== null) {
+      if (images.length < 3) {
+        setImages([...images, "data:image/jpeg;base64," + result["base64"]]);
+      } else {
+        let updatedImages = [...images];
+        updatedImages[2] = "data:image/jpeg;base64," + result["base64"];
+        setImages(updatedImages);
+      }
```

Comment:
Why do we need this if/else logic? We're only rendering the button that calls this function if `images.length < 3`, so we shouldn't need this check right

Reply:
Ohhh shoot, I forgot to delete that part. I was originally gonna use different logic for editing images but forgot to delete the old code